src/Raven.Client/Documents/Indexes/IndexDefinition.cs

```diff
@@ -98,6 +98,8 @@ public virtual IndexSourceType SourceType
         }
 
         public virtual ArchivedDataProcessingBehavior? ArchivedDataProcessingBehavior { get; set; }
+        
+        public string SchemaValidation { get; set; }
```

Comment:
internal?

Also shouldn't it be SchemaValidationConfiguration, exactly what we have in the DatabaseRecord? So we can handle multiple collections?